src/validation.h

```diff
@@ -18,6 +18,7 @@
 #include <kernel/chainparams.h>
 #include <kernel/chainstatemanager_opts.h>
 #include <kernel/cs_main.h> // IWYU pragma: export
+#include <kernel/disconnected_transactions.h>
```

Comment:
I think this should be included in `validation.cpp` instead.

Reply:
moved